XenAdmin/Wizards/NewPolicyWizard/NewPolicyWizard.cs

```diff
@@ -41,31 +41,200 @@
 
 namespace XenAdmin.Wizards.NewPolicyWizard
 {
-    // This class acts as the base class for NewPolicyWizardSpecific. It's only here
-    // because of a bug in Visual Studio: the Designer can't design classes of a
-    // generic class. The workaround is to do the design in this non-generic class,
-    // and then inherit the generic class from it. See
-    // http://stackoverflow.com/questions/1627431/fix-embedded-resources-for-generic-usercontrol
-    // http://bytes.com/topic/c-sharp/answers/537310-can-you-have-generic-type-windows-form
-    // http://connect.microsoft.com/VisualStudio/feedback/details/115397/component-resource-manager-doesnt-work-with-generic-form-classes
-    // (or search on Google for [ComponentResourceManager generic]).
-
     public partial class NewPolicyWizard : XenWizardBase
     {
         protected NewPolicyPolicyNamePage xenTabPagePolicy;

```

Comment:
Do these pages have to be protected? I'd rather not have wizard controls exposed if we can avoid it, because nothing prevents people from calling them and creating complicated code.